web/src/components/steps/certificateOfService/reviewCSV/pdf/Form7Layout.vue

```diff
@@ -228,7 +219,7 @@
                     :shiftmark="1"
                     boxMargin="0" 
                     style="display:inline;"
-                    :check="documentList.includes('Application About Priority Parenting Matter')?'yes':''" text="Application About Priority Parenting Matter"/>
+                    :check="documentList.includes('Application About a Protection Order')?'yes':''" text="Application About a Protection Order"/>
```

Comment:
Are these values changed in the json as well ?

Reply:
Not yet. I am having separate PR for form changes. This covers only the layout.